raiden/smart_contracts/NettingChannelContract.sol

```diff
@@ -102,7 +102,9 @@ contract NettingChannelContract {
     /// @param secret The secret to unlock the locked transfer.
     function unlock(bytes locked_encoded, bytes merkle_proof, bytes32 secret) {
         data.unlock(msg.sender, locked_encoded, merkle_proof, secret);
-        ChannelSecretRevealed(secret);
+
+        // Note: this is only valid as long as msg.sender is one of the participants
```

Comment:
I see. Yes indeed `unlock()` will throw if msg.sender is not a participant.
@hackaugusto Can you perhaps change the comment to reflect that? Just say that we can't get here without `msg.sender` being a participant due to the `unlock` function?